### PR TITLE
Extract cheatsheet into separate package

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
## Checklist

- [/] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [/] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [/] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-cheatsheet/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "target": "es2020",
```
Comment (pokey) [inline packages/cursorless-cheatsheet/tsconfig.json:4]: try to remove

Comment (AndreasArvidsson) [reply packages/cursorless-cheatsheet/tsconfig.json]: removed!

File: packages/cursorless-cheatsheet/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "exports": {
+    ".": {
+      "cursorless:bundler": "./src/index.ts",
+      "default": "./out/index.js"
+    }
+  },
+  "dependencies": {
+    "@cursorless/common": "workspace:*",
+    "immer": "^10.0.4",
+    "lodash": "^4.17.21",
```
Comment (pokey) [inline packages/cursorless-cheatsheet/package.json:29]: don't forget to update this to lodash-es now that #2476 has merged

File: packages/cursorless-cheatsheet/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "node-html-parser": "^6.1.12"
+  },
+  "devDependencies": {
+    "@types/lodash": "4.17.0"
```
Comment (pokey) [inline packages/cursorless-cheatsheet/package.json:33]: here too

File: packages/cursorless-cheatsheet/src/Cheatsheet.ts [hunk 4]
```diff
@@ -27,16 +26,15 @@ interface CheatSheetCommandArg {
...
+export async function showCheatsheet(
+  ide: IDE,
+  { version, spokenFormInfo, outputPath }: CheatSheetCommandArg,
+) {
   if (version !== 0) {
     throw new Error(`Unsupported cheatsheet api version: ${version}`);
   }
 
-  const cheatsheetPath = path.join(ide().assetsRoot, "cheatsheet.html");
+  const cheatsheetPath = path.join(ide.assetsRoot, "cheatsheet.html");
```
Comment (pokey) [inline packages/cursorless-cheatsheet/src/Cheatsheet.ts:37]: this is better 👍

**Review by pokey — COMMENTED** [review]
good stuff; just needs lodash-es fixes

File: packages/cursorless-cheatsheet/package.json (hunk 2 above)
Comment (AndreasArvidsson) [reply packages/cursorless-cheatsheet/package.json:29]: done

File: packages/cursorless-cheatsheet/package.json (hunk 3 above)
Comment (AndreasArvidsson) [reply packages/cursorless-cheatsheet/package.json]: done

**Review by pokey — APPROVED** [review]

**Pull request merged**